### DISCUSSION TITLE
Error 500 only on local timeline since today [help]

### DISCUSSION THREAD

**Discussion by r3k2** [mastodon/mastodon#18287] [Q&A]
https://hispagatos.space/api/v1/timelines/public?local=true&only_media=false
I was on 3.5.0rc1 and this started to happen, to see if it will fix I upgraded one by one all the way to 3.5.1 latest one. but is still doing it. if someone posts right at the moment it shows up on the local timeline but if I reload the browser it disappears.. The link above is the one getting 500 errors that I got from developer tools.
Not sure why is doing this..

**Comment by ClearlyClaire** [comment]
Hi! Anything in the logs? (`journalctl -u mastodon-web`)

**Comment by r3k2** [comment]
This is weird, I just woke up and seems to  be working now... O_O this is good but at the same time, makes me nervous.

**Reply by ClearlyClaire** [reply]
You should still have backtraces in the logs.

**Reply by r3k2** [reply]
I will check in a bit, on my way to work now.
